watertap/examples/flowsheets/RO_with_energy_recovery/monte_carlo_sampling_RO_ERD.py

```diff
@@ -101,6 +100,56 @@ def run_parameter_sweep(
     outputs["EC"] = m.fs.costing.specific_energy_consumption
     outputs["LCOW"] = m.fs.costing.LCOW
 
+    ParameterSweep._assign_variable_names(m, outputs)
```

Comment:
Why are we calling an private class method here?

Reply:
I did this because it needs to happen after the outputs are built, which happens inside the rebuilding function. I'll see if it can be cleaned up or removed.